### PR TITLE
Check only for one drbd resource when picking first node for upgrade

### PR DESCRIPTION
**Pull request by jsuchome**
When both resources (postgres, rabbitmq) have different allocations
of master/slave nodes, previous call would not be able to find
the slave node.

### CODE REVIEW

**Comment by jsuchome** [comment]
[(13) S50P6 (S49P16) Orchestrate upgrade of controller nodes in HA setup (upgrade step 12 part 2 -- HA/DRBD/1cluster)](https://trello.com/c/2PMGauJx/118-13-s50p6-s49p16-orchestrate-upgrade-of-controller-nodes-in-ha-setup-upgrade-step-12-part-2-ha-drbd-1cluster)

File: crowbar_framework/app/models/api/upgrade.rb [hunk 1]
```diff
@@ -257,7 +257,7 @@ def upgrade_controller_nodes
         NodeObject.find(
           "state:crowbar_upgrade AND (roles:database-server OR roles:rabbitmq-server)"
         ).each do |db_node|
-          cmd = "LANG=C crm resource status ms-drbd-{postgresql,rabbitmq}\
+          cmd = "LANG=C crm resource status ms-drbd-postgresql\
```
Comment (rhafer) [inline crowbar_framework/app/models/api/upgrade.rb]: @jsuchome postgresql and rabbitmq could run in different clusters. With this change we would not find a node on the cluster just running rabbitmq. 
It also possible I think to configure one of them to use drbd but the other one for a different type of shared storage.

That being said, it seem this function seems to have some other issues if there are multiple drbd clusters. So maybe we should handle that in a separate PR/Trello card?

Comment (jsuchome) [inline crowbar_framework/app/models/api/upgrade.rb]: @rhafer what about this - take only one resource that is actually running there:

`LANG=C crm resource status ms-drbd-{postgresql,rabbitmq} | sort | head -n 2 | grep \\$(hostname)`

Comment (rhafer) [inline crowbar_framework/app/models/api/upgrade.rb]: Yeah, makes sense

**Pull request merged**